articles/docker.md

```diff
@@ -87,4 +83,9 @@ Docker tags can be assigned to later commits. If you want to be explicit about t
 docker pull docker.synapse.org/syn12345/mytestrepo@sha256:2e36829f986351042e28242ae386913645a7b41b25844fb39b29af0bdf8dcb63
 ```
 
-where the digest for a commit is printed to the command line after a successful Docker push. The Synapse web portal displays current digests for a repository's tags on the repository's Synapse page.
\ No newline at end of file
+where the digest for a commit is printed to the command line after a successful Docker push. The Synapse web portal displays current digests for a repository's tags on the repository's Synapse page.
+
+{% include note.html content="You can add external repositories, i.e. repositories that live in other registries like DockerHub and quay.io. For these repositories there is no tight integration (Synapse doesn't contact these external registries) but it allows you to list Docker repositories that are relevant to the project but are not within Synapse.
```

Comment:
I think you mean 'e.g.' not 'i.e.'. It should also should have a trailing comma (i.e., like 'e.g.,').